manifests/init.pp

```diff
@@ -136,7 +98,7 @@
     }
   }
 
-  file { "${settings::confdir}/splunk_hec.yaml":
+  file { '/etc/puppetlabs/puppet/splunk_hec.yaml':
```

Comment:
Is this part of common events? It seems like basing the config file off the confdir is good either way.

Reply:
Yeah, we can do this here.